bin/ch-run.c

```diff
@@ -36,7 +36,8 @@ Example:\n\
   $ ch-run /data/foo -- echo hello\n\
   hello\n\
 \n\
-You cannot use this program to actually change your UID.";
+You cannot use this program to actually change your UID.\n\
+\n";
```

Comment:
Why newlines added?

Reply:
Also to look similar to the other messages.